app/lib/features/events/dialogs/create_event_sheet.dart

```diff
@@ -44,15 +44,16 @@ class _CreateEventSheetConsumerState extends ConsumerState<CreateEventSheet> {
   void initState() {
     super.initState();
     Future(() {
```

Comment:
Same goes here. Replace ```Future()``` with ```WidgetsBinding.instance.addPostFrameCallback```. Please check if this is happening more than few files and update it.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/855/commits/81b669d08592535b7e6344fdaea9d2ecaa89c08d